cabot/rest_urls.py

```diff
@@ -124,33 +125,32 @@ class ViewSet(viewset_class):
     ),
 ))
 
-'''
-Omitting user API, could expose/allow modifying dangerous fields.
+# User API is off by default, could expose/allow modifying dangerous fields
+if settings.EXPOSE_USER_API:
+    router.register(r'users', create_viewset(
+        arg_model=django_models.User,
+        arg_fields=(
+            'password',
+            'is_active',
+            'groups',
+            #'user_permissions', # Doesn't work, removing for now
+            'username',
+            'first_name',
+            'last_name',
+            'email',
+        ),
+    ))
 
-router.register(r'users', create_viewset(
-    arg_model=django_models.User,
-    arg_fields=(
-        'password',
-        'is_active',
-        'groups',
-        #'user_permissions', # Doesn't work, removing for now
-        'username',
-        'first_name',
-        'last_name',
-        'email',
-    ),
-))
+    router.register(r'user_profiles', create_viewset(
+        arg_model=models.UserProfile,
+        arg_fields=(
+            'user',
+            'mobile_number',
+            'hipchat_alias',
```

Comment:
@dbuxton OK... can you elaborate on that or point me in the right direction?  I'd be happy to make any changes necessary to address your concern.

FWIW, it's working for me and I'm using at least one 3rd party plugin.  I tested it with the setting on and off.